components/esp32/Kconfig

```diff
@@ -929,6 +929,25 @@ menu "ESP32-specific"
             This option depends on the CONFIG_FREERTOS_UNICORE option because RTC fast memory
             can be accessed only by PRO_CPU core.
 
+config USE_FIXED_STATIC_RAM_SIZE
+	bool "Use fixed static RAM size"
+	default n
```

Comment:
There is a mix of tabs and spaces here (tab on this line and spaces below). To be sure that the format of this Kconfig file is correct, please run `tools/check_kconfigs.py --directory components/esp32`.

Reply:
fixed.